video/out/cocoa/events_view.m

```diff
@@ -35,12 +35,40 @@ @implementation MpvEventsView
 @synthesize tracker = _tracker;
 @synthesize hasMouseDown = _mouse_down;
 
+#if !HAVE_MACOS_10_13_FEATURES
+typedef NSString* NSPasteboardType;
+#endif
+
+static NSPasteboardType filenameType(void)
+{
+#if HAVE_MACOS_10_13_FEATURES
+    if (@available(macOS 10.13, *)) {
+        return NSPasteboardTypeFileURL;
+    } else
+#endif
+    {
+        return (NSPasteboardType)kUTTypeFileURL;
+    }
+}
+
+static NSPasteboardType urlType(void)
+{
+#if HAVE_MACOS_10_13_FEATURES
+    if (@available(macOS 10.13, *)) {
+        return NSPasteboardTypeURL;
+    } else
```

Comment:
the else case here seems redundant, `(NSPasteboardType)kUTTypeURL` can be returned directly. same for the function above.